tuner/tuner/candidate_gen_test.py

```diff
@@ -77,20 +77,21 @@ def test_get_td_spec_contraction(tuner_ctx: common.TunerContext) -> None:
         subgroup_n_count=16,
     )
     pipeline_attr = iree_codegen.DispatchLoweringPassPipelineAttr.get(
-        iree_codegen.DispatchLoweringPassPipeline.LLVMGPUVectorDistribute
+        iree_codegen.DispatchLoweringPassPipeline.LLVMGPUVectorDistribute,
+        ctx=tuner_ctx.mlir_ctx,
     )
     pipeline_options = iree_gpu.PipelineOptionsAttr.get(prefetch_shared_memory=True)
     config_dict = common.get_translation_info_config(pipeline_options, waves_per_eu=8)
     translation_info = iree_codegen.TranslationInfoAttr.get(
-        pipeline_attr, None, [16, 16, 1], 16, config_dict
+        pipeline_attr, None, [16, 16, 1], 16, config_dict, ctx=tuner_ctx.mlir_ctx
     )
     compilation_info = iree_codegen.CompilationInfoAttr.get(
-        lowering_config, translation_info
+        lowering_config, translation_info, ctx=tuner_ctx.mlir_ctx
```

Comment:
I don't think we need this or similar elsewhere -- `None` context is always replaced with the first context in the context stack: https://mlir.llvm.org/docs/Bindings/Python/#context-management